components/StyledTabs.tsx

```diff
@@ -51,19 +53,20 @@ type TabsProps = {
   onChange?: Function;
 };
 
-const TabsBar = styled(Flex)`
-  border: 1px solid transparent;
+const TabsBar = styled.div`
   border-bottom: 1px solid ${props => props.theme.colors.black[300]};
 `;
 
 const Tabs = ({ tabs, selectedId, onChange, ...props }: TabsProps & Parameters<typeof Flex>[0]) => {
   return (
-    <TabsBar {...props} gap="12px">
-      {tabs.map(tab => (
-        <TabButton key={tab.id} onClick={() => onChange?.(tab.id)} selected={tab.id === selectedId}>
-          {tab.label} {tab.count && <span>{tab.count}</span>}
-        </TabButton>
-      ))}
+    <TabsBar {...props}>
+      <Flex gridGap="24px" mb="-1px" overflowX="auto">
```

Comment:
What is the benefit of having two components instead of a single `styled(Flex)`?

Reply:
There was some combination of getting the negative margin of 1px (to have the tab buttons border be on top of the tabsbar border) and the overflowX="auto" that created a vertical scroll when the tabs overflowed in the horizontal direction